### PR TITLE
Move show sentence into admin

### PR DESCRIPTION
**Pull request by darionyaphet**

### CODE REVIEW

File: src/parser/AdminSentences.cpp [hunk 1]
```diff
@@ -24,6 +24,46 @@ std::string ShowPartsSentence::toString() const {
     return std::string("SHOW PARTS");
 }
 
+std::string ShowTagsSentence::toString() const {
```
Comment (laura-ding) [inline src/parser/AdminSentences.cpp:27]: Schema is belonged to maintain

Comment (darionyaphet) [reply src/parser/AdminSentences.cpp:27]: Actually I want to move `show sentence` into one place.

Comment (laura-ding) [reply src/parser/AdminSentences.cpp:27]: The classification is consistent with the validate and executor. I don’t think it needs to move.

Comment (darionyaphet) [reply src/parser/AdminSentences.cpp:27]: Emmm ... The core of this problem is that what do you think about `show` sentence. 

IF you think it is a admin sentence, you should put them together. 

According to you, `ShowUsersSentence` should put into `UserSentence`

Comment (laura-ding) [reply src/parser/AdminSentences.cpp:27]: Actual the `ShowUsersSentence` also belongs to the `admin`. Everything related to the user belongs to `Admin`.

**Comment by yixinglu** [comment]
Now plan nodes are classified as following:
**admin**:
- user
- role
- space
- host
- snapshot

**maintain**:
- tag
- edge
- index

I agree with @laura-ding, if we need to change this classified method later, we should also update validator and executor related codes. 

Now i close the PR, If we need it, reopen it later. Thanks for your efforts!

**Pull request closed without merging**